src/hotspot/share/runtime/globals.hpp

```diff
@@ -1996,6 +1996,9 @@ const int ObjectAlignmentInBytes = 8;
       "Mininal PID value for checkpoint'ed process")                        \
       range(1, UINT_MAX)                                                    \
                                                                             \
+  product(bool, CRaCResetStartTime, false, RESTORE_SETTABLE,                    \
+      "Reset JVM's start time and uptime on restore")                     \
+                                                                            \
```

Comment:
Sorry for being late. Since we're evaluating providing the restore time via getUptime(), the option should be true by default, and be diagnostic.

Reply:
Ok, I will change it in a new PR